completions/cppcheck

```diff
@@ -42,7 +42,9 @@ _comp_cmd_cppcheck()
             return
             ;;
         -j)
-            COMPREPLY=($(compgen -W "{2..$(_ncpus)}" -- "$cur"))
+            local ret
```

Comment:
An unrelated note, but now that we're sprinkling `local ret` quite a bit and more to come, perhaps adding it to the standard `local cur prev words cword was_split comp_args` everywhere would not be a bad idea.

Reply:
This is just my preference, but I'd like to keep the variable scope small generally. Maybe this results in possibly multiple declarations of `ret` in a function, but I kind of accepted and got used to it because it doesn't cause a problem in Bash.

Including it in the standard variable list might be useful to prevent the variable leak in the case where one forgets to declare `ret` in calling a function that sets `ret`. But in another way of thinking, that hides the bug that one forgot to declare `ret`. I cannot decide which one is better for this point.